internal/cluster/transition.go

```diff
@@ -43,6 +44,15 @@ type transitionHandler struct {
 	installationTimeout time.Duration
 	finalizingTimeout   time.Duration
 	eventsHandler       eventsapi.Handler
+	usageAPI            usage.API
+}
+
+func (th *transitionHandler) setUsage(enabled bool, name string, props *map[string]interface{}, usages map[string]models.Usage) {
```

Comment:
I think i don't fully understand the need for this function, it is used one time with enabled=true, so why we need it?
and not just using "th.usageAPI.Add(usages, name, props)"

Reply:
That's a fair point, though a suggestion from Ori negates the need to do this at all, so I will be changing this completely.